src/navigation/main/MainTabNavigator.tsx

```diff
@@ -20,34 +25,35 @@ const MainTab = createBottomTabNavigator()
 
 const MainTabs = () => {
   const { white, grayLight } = useColors()
-  const navigation = useNavigation()
+  const navigation = useNavigation<RootNavigationProp>()
+  const tabNavigation = useNavigation<MainTabNavigationProp>()
   const {
     app: { isLocked, isSettingUpHotspot },
   } = useSelector((state: RootState) => state)
-  const notification = useSelector(
-    (state: RootState) => state.notifications.notification,
+  const pushNotification = useSelector(
+    (state: RootState) => state.notifications.pushNotification,
   )
   const dispatch = useAppDispatch()
 
   useEffect(() => {
     if (!isLocked) return
-    navigation.navigate('LockScreen', { requestType: 'unlock', lock: true })
+    navigation.push('LockScreen', { requestType: 'unlock', lock: true })
```

Comment:
Just curious why these were changed from navigate to push?

Reply:
Thats what they were originally, I was actually unsure on the difference between the two but I decided to put them back to what they were previously.

Reply:
Ah looks like the only difference is push allows the component to be pushed on the stack multiple times. In the case they have the same effect